tests/subsys/usb/bos/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
   usb.bos:
-    platform_whitelist: native_posix
+    platform_whitelist: native_posix frdm_k64f
```

Comment:
I guess it should only be tested on native_posix, @finikorg ?

Reply:
@jfischer-phytec-iot yes.

Reply:
Why?

Reply:
Those (bos and osdesc) are unit tests, they are not "build tests" which only need to be built but run tests, they need to be executed on the host.

Reply:
With `--device-testing`, they **are** executed on the host. In this case, the host is frdm_k64f.

Reply:
that's good, I'm for it